waves/wallet/Cargo.toml

```diff
@@ -13,7 +13,8 @@ default = [ "console_error_panic_hook" ]
 [dependencies]
 aes-gcm-siv = { version = "0.9", features = [ "std" ] }
 anyhow = "1"
-bdk = { version = "0.2", default-features = false }
+# bdk = { version = "0.2", default-features = false }
+bdk = { git = "https://github.com/bitcoindevkit/bdk", default-features = false }
```

Comment:
I thought I give it a try to see if things work with the latest master of bdk. I know it is not quite in the spirit of "remove patched dependencies" to then add another git dependency.

We can also wait until it is actually released, I don't mind much although it is nice to know that we no longer depend on our forks.

Reply:
I'm okay with this!